pages/windows/reg-flags.md

```diff
@@ -0,0 +1,19 @@
+# reg flags
+
+> Display or set flags on registry keys.
+
+- Display current flags for a specific key:
+
+`reg flags {{key_name}} query`
+
+- Display help and available flag types:
+
+`reg flags /?`
+
+- Set specified flags, and unset unmentioned flags, for a specific key:
```

Comment:
How do you mention multiple flags ? Is there a separator ?

Reply:
Just a space between flags. I'll mention that.